src/Resources/config/core.xml

```diff
@@ -55,6 +56,12 @@
                 <argument type="service" id="sonata.admin.breadcrumbs_builder"/>
             </call>
         </service>
+        <service id="Sonata\AdminBundle\Translator\Extractor\AdminExtractor">
+            <tag name="translation.extractor" alias="sonata_admin"/>
+            <argument type="service" id="sonata.admin.pool"/>
+            <argument type="service" id="sonata.admin.breadcrumbs_builder"/>
+            <argument type="service" id="logger" on-invalid="ignore"/>
```

Comment:
The `logger` parameter is missing in the constructor signature

Reply:
thanks! done